package.json

```diff
@@ -35,13 +35,15 @@
     "karma-chrome-launcher": "^2.2.0",
     "preprocessor": "^1.4.0",
     "sinon": "^5.0.10",
-    "uglify-js": "^3.3.26"
+    "uglify-js": "^3.3.26",
+    "local-web-server": "2.6.0"
```

Comment:
Do you really want to add this? e.g. I use `http-server`and probably people are using their own servers.

Reply:
Actually, I think 'serve' is the most widely used package - I'd use that: https://www.npmjs.com/package/serve